test/onnx/test_fx_op_consistency.py

```diff
@@ -648,6 +654,14 @@ def skip_torchlib_forward_compatibility(
         matcher=lambda sample: sample.kwargs.get("include_self") is False,
         reason="ONNX does't support include_self=False option",
     ),
+    xfail(
+        "sub",
+        matcher=lambda sample: sample.input.dtype
```

Comment:
Use the dtype argument to specify dtypes? Should we test these types at all or should we skip them globally?

Reply:
thanks, I agree they should be moved up.